release-notes.md

```diff
@@ -0,0 +1,60 @@
+# Release v0.9.0
+
+## New Features
```

Comment:
>Features [](start = 7, length = 8)

* new REST APIs to update Virtual Cluster
* Support training on RDMA network for on-premise clusters